builder/builder.go

```diff
@@ -126,7 +127,8 @@ func NewFromConfig(conf string) (*Builder, error) {
 
 // InitMix will initialise a new swupd-client consumable "mix" with the given
 // based Clear Linux version and specified mix version.
-func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allUpstream bool, noDefaults bool, upstreamURL string, git bool) error {
+func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allUpstream bool, noDefaults bool,
```

Comment:
I'm not sure I understand this. If the user does
`mixer init --upstream-bundles-url=<url>`, shouldn't this <url> value get saved in the new builder.conf parameter Swupd.UPSTREAM_BUNDLES_URL.

Reply:
Yes, but see how we do it for `format` in `mixer.state`:
https://github.com/clearlinux/mixer-tools/blob/7a776cbb665986d7215e7c7b14dd32c401701bb6/mixer/cmd/init.go#L51

We should follow similar approach here. Since config carries the information and is part of the builder object that is calling `InitMix` we can set it before the call instead of passing it as a parameter.

Reply:
> Lastly, please run our automated tests in your PR. 1 of the 3 failures reported by Travis is due to problems in the PR itself.
I was only running `make check`. John just reminded me of `make batcheck`. Will run that too.